uftrace.h

```diff
@@ -30,6 +30,9 @@
 #define KB 1024
 #define MB (KB * 1024)
 
+#define stringify(s)    __stringify(s)
+#define __stringify(s)  #s
+
```

Comment:
Could you please move them to "utils.h"?

Reply:
done.